tasks/ebpf.py

```diff
@@ -91,19 +91,37 @@ def cleanup_verifier_stats(verifier_stats):
         "jsonfmt": "Output in json format rather than tabulating",
         "out": "Output file to write results to. By default results are written to stdout",
         "debug_build": "Collect verification statistics for debug builds",
-    }
+        "filter_file": "List of files to load ebpf program from, specified without extension. By default we load everything",
+        "grep": "Regex to filter program statistics",
+    },
+    iterable=['filter_file', 'grep'],
 )
-def print_verification_stats(ctx, skip_object_files=False, base=None, jsonfmt=False, out=None, debug_build=False):
+def print_verification_stats(
+    ctx,
+    skip_object_files=False,
+    base=None,
+    jsonfmt=False,
+    out=None,
+    debug_build=False,
+    filter_file=None,
+    grep=None,
+):
     sudo = "sudo -E" if not is_root() else ""
     if not skip_object_files:
         build_object_files(ctx)
         build_cws_object_files(ctx)
 
     ctx.run("go build -tags linux_bpf pkg/ebpf/verifier/calculator/main.go")
 
-    debug = "--debug" if debug_build else ""
     env = {"DD_SYSTEM_PROBE_BPF_DIR": "./pkg/ebpf/bytecode/build"}
-    res = ctx.run(f"{sudo} ./main {debug}", env=env, hide='out')
+    args = (
+        [
+            "--debug" if debug_build else "",
+        ]
+        + [f"-filter-file {f.split('.')[0]}" for f in filter_file]
```

Comment:
This way will trim only the last `.ext`:
```suggestion
        + [f"-filter-file {os.path.splitext(f)[0]}" for f in filter_file]
```

Reply:
Removing this and adding a separate loop to verify that the extension is valid, i.e. `.o`